libtiledbvcf/src/dataset/tiledbvcfdataset.cc

```diff
@@ -270,7 +270,8 @@ void TileDBVCFDataset::create(const CreationParams& params) {
 
   // Log the group structure
   group.open(TILEDB_READ);
-  LOG_DEBUG("TileDB Groups: \n{}", group.dump(true));
+  // NOTE: On windows group.dump produces memory access violation
+  // LOG_DEBUG("TileDB Groups: \n{}", group.dump(true));
```

Comment:
I would have entirely removed it, or condition it on debug-level logs being enabled (we don't have an `is_enabled` global function). Dumping groups is quite expensive as it involves several group open and I/O operations.